Assets/HoloToolkit/Boundary/Scripts/SceneContentAdjuster.cs

```diff
@@ -7,40 +10,82 @@
 using UnityEngine.VR;
 #endif
 
-public class SceneContentAdjuster : MonoBehaviour
+namespace HoloToolkit.Unity.Boundary
 {
-    private Vector3 lastFloorHeight;
-    private float floorHeightOffset = 1f;
-
-    private void Awake()
+    public class SceneContentAdjuster : MonoBehaviour
     {
-#if UNITY_2017_2_OR_NEWER
-        if (Application.isEditor && XRDevice.isPresent)
+        private int frameWaitHack = 0;
+
+        [SerializeField]
+        [Tooltip("Optional container object reference. If null, this script will move the object it's attached to.")]
+        private Transform containerObject;
+
+        [SerializeField]
+        [Tooltip("Select this if the container should be placed in front of the head on app launch in a room scale app.")]
+        private bool alignWithHeadHeight = true;
+
+        [SerializeField]
+        [Tooltip("Use this to set the desired position of the container in a stationary app.")]
+        private Vector3 stationarySpaceTypePosition = Vector3.zero;
```

Comment:
These I think will cause more confusion than solve.
Perhaps more text around how or when to tweak them would be good.